### ISSUE TITLE
search button missing text/icon on mobile

### ISSUE THREAD

**Issue by iamjolly** [18F/cv_faq#171]
On category detail pages, mobile view...

It looks like we're positioning the search button text off-screen when viewport width is less than 30ems. 

<img width="408" alt="Screen Shot 2020-03-20 at 6 40 05 AM" src="https://user-images.githubusercontent.com/1093423/77165425-e42c9180-6a77-11ea-8286-1848bbcd551b.png">

This may be a known issue already being addressed. If so, please close.
